src/components/modals/StaticModalComponent.js

```diff
@@ -37,20 +37,17 @@ class StaticModalComponent extends Component<Props> {
   render () {
     const styles = StaticModalStyle
     return (
-      <Modal style={styles.container} animationType={'slide'} transparent visible>
-        <TouchableOpacity style={styles.touchOut} onPress={this.props.cancel}>
+      <Modal style={styles.container} animationType={'slide'} isVisible={this.props.isVisible}>
+        <TouchableOpacity onPress={this.props.cancel}>
           <View style={styles.modalBox}>
-            <LinearGradient
-              style={styles.header}
-              start={{ x: 0, y: 0 }}
-              end={{ x: 1, y: 0 }}
-              colors={[THEME.COLORS.GRADIENT.DARK, THEME.COLORS.GRADIENT.LIGHT]}
-            >
-              <Icon style={styles.icon} name={Constants.CHECK_CIRCLE} size={styles.iconSize} type={Constants.SIMPLE_ICONS} />
-            </LinearGradient>
-            <View style={styles.bottom}>
-              <View style={styles.bodyRow}>
-                {this.renderMiddle(styles)}
+            <View style={styles.innerBox}>
+              <Gradient style={styles.header}>
```

Comment:
Please go back to using Linear Gradient instead of Gradient component. The reason the component was not used is that for the static modals the gradient goes in reverse of the header so from left to right it should be dark to light instead of light to dark like the header. The other option is to refactor the Gradient component to take an argument for the direction of the gradient.